src/qml/NavigationHighlight.qml

```diff
@@ -0,0 +1,34 @@
+import QtQuick 2.12
+
+import org.qgis 1.0
+import org.qfield 1.0
+
+import Theme 1.0
+
+Item {
+  id: navigationHighlight
+  property Navigation navigation
+
+  LinePolygonHighlight {
+    visible: positionSource.active
+    mapSettings: navigation.mapSettings
+    geometry:   QgsGeometryWrapper {
+      qgsGeometry: navigation.path
+      crs: navigation.mapSettings.crs
+    }
+    color: "#77FF0000"
```

Comment:
Can you add this color to the theme and/or name it?

Reply:
done